pages/about.md

```diff
@@ -1,47 +1,27 @@
 ---
 layout: page
-title: What is Wax?
+title: About the Exhibit
```

Comment:
I would change this to "About the Project" or something and not refer to the whole site as an exhibit - it's a site or project with multiple exhibits? Just my 2 cents, feel free to ignore. And this shouldn't hold up a merge, you can always update it later.